requirements/pi.txt

```diff
@@ -16,6 +16,7 @@ onnxruntime!=1.16.0  # TODO: Patching https://github.com/microsoft/onnxruntime/i
 ovos-ww-plugin-precise~=0.1
 ovos-ww-plugin-precise-lite[tflite]~=0.1,>=0.1.2
 ovos-ww-plugin-vosk~=0.1,>=0.1.1
+openwakeword~=0.5,<0.6
```

Comment:
Can you add a comment for why this is here? Hopefully it can be removed soon when fixed upstream